src/rules/selector-max-specificity/__tests__/index.js

```diff
@@ -177,3 +181,25 @@ testRule(rule, {
     description: "ignore rules with variable interpolation",
   }],
 })
+
+test("Cannot Parse Selector Warning", t => {
+  const config = {
+    rules: {
+      "selector-max-specificity": ["0,2,0"],
+    },
+  }
+  let planned = 0
+
+  postcss().use(stylelint(config)).process(".a, :matches(.b, .c, .d) e:first-child, .f > :patches(:first-child, :last-child) { }").then(result => {
```

Comment:
I don't think we need to use PostCSS JS API here. We can just use stylelint's, [like so](https://github.com/stylelint/stylelint/blob/master/src/rules/declaration-block-properties-order/__tests__/validate-options.js#L1-L23). Keep your `let planned = 0` approach though.